README.md

```diff
@@ -136,12 +136,12 @@ Do not update keys with no comments next to them. For your job, do not copy from
         env:
           - name: ADDON_IDS
             value: reference-addon // update to your addons
-          - name: ADDON_TEST_HARNESSES
+          - name: TEST_HARNESSES
             value: quay.io/rmundhe_oc/osde2e-example-test-harness // update to your test harness image
           - name: CHANNEL
```

Comment:
```
      - name: POLLING_TIMEOUT
        value: "7200"
```

@ritmun I think we should add this env var to the example. If someone was to take this and make a copy/modify, their test would then only run for 5 minutes.